src/components/CloudHeader/CloudHeader-test.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import CloudHeader from './CloudHeader';
+import CloudHeaderWrapper from './CloudHeaderWrapper';
+import CloudHeaderMenu from './CloudHeaderMenu';
+import CloudHeaderLogo from './CloudHeaderLogo';
+import CloudHeaderList from './CloudHeaderList';
+import CloudHeaderListItem from './CloudHeaderListItem';
+import { shallow, mount } from 'enzyme';
+
+describe('CloudHeader', () => {
+  describe('Renders as expected', () => {
```

Comment:
Should we just use snapshot tests here to validate the `bx` classes?

Reply:
Sure, how do we go about integrating that in?

Reply:
Oh shoot, is that in this add-on yet?